static/css/cdr_common.css

```diff
@@ -1,382 +1,382 @@
 body {
-	text-align: center;
-	font-family: verdana, sans-serif;
-	background-color: #ececec;
-	color: #222;
-	font-size: 12px;
-	line-height: 17px;
+    text-align: center;
```

Comment:
Ahh, all the indent characters in the css files changed. If its not too much of a pain, could you revert that and we'll make that change in another PR? Otherwise it will be hard to identify what changes were involved in updating to the new design in the future.

Does your IDE have a setting for only reformatting lines which are modified?